addons/l10n_in/data/template/account.tax-in.csv

```diff
@@ -48,6 +48,10 @@
 "","","","","","","","","","","","","","","","","","","tax","invoice","p11234","l10n_in.tax_tag_igst","","","",""
 "","","","","","","","","","","","","","","","","","","base","refund","","l10n_in.tax_tag_base_igst","","","",""
 "","","","","","","","","","","","","","","","","","","tax","refund","p11234","l10n_in.tax_tag_igst","","","",""
+"igst_sale_40","40% IGST S","IGST","IGST 40%","sale","gst","percent","40.0","","False","igst_group","False","True","","","","","","base","invoice","","l10n_in.tax_tag_base_igst","","","sgst_sale_40",""
```

Comment:
@masm-odoo @ajjo-odoo Is this the order you want it to have?  Because as we do not put the sequence this one is going to come after the 5% in the interface?  In the file itself, it is a bit weird that it comes after the 5%, while the others are sorted.